raft/fsm.hh

```diff
@@ -47,7 +47,7 @@ struct fsm_output {
         return !term_and_vote &&
             log_entries.size() == 0 && messages.size() == 0 &&
             committed.size() == 0 && !snp && snps_to_drop.empty() &&
-            !configuration;
+            !configuration && !max_read_id_with_quorum;
```

Comment:
Please update the patch changelog with an explanation about why this is needed.

Reply:
@avikivity, done! pls let me know if anything else needs change.